src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -324,7 +324,7 @@ private void OnPropertyChanged(object sender, PropertyChangedEventArgs propertyC
             }
         }
 
-        protected virtual void OnIsolationModeRequestUpdate()
+        protected virtual void OnIsolationModeRequestUpdate(bool? value = null)
```

Comment:
Calling `OnIsolationModeRequestUpdate()` will only make a trigger to update the preview without necessarily changing the toggle value.